cpp/modmesh/universe/World.hpp

```diff
@@ -75,6 +77,25 @@ class World
     World & operator=(World &&) = delete;
     ~World() = default;
 
+    void add_vertex(vertex_type const & vertex);
```

Comment:
API for vertices:

* `add_vertex()`
* `nvertex()`
* `vertex()` (accessor without bound check)
* `vertex_at()` (accessor with bound check)